conf/broker.conf

```diff
@@ -1566,6 +1566,10 @@ exposePublisherStats=true
 statsUpdateFrequencyInSecs=60
 statsUpdateInitialDelayInSecs=60
 
+includeHealthCheckInMetrics=true
+healthCheckFrequencyInSeconds=60
+healthCheckInitialDelayInSecs=60
```

Comment:
instead, all 3 config variables just keep one variable: `healthCheckMetricsUpdateTimeInSeconds=-1`. here `-1` means disable. and please add config documentation in config.

Reply:
addressed